diff --git a/packages/client/src/components/signup.vue b/packages/client/src/components/signup.vue
index 58c15d81b..51af998e7 100644
--- a/packages/client/src/components/signup.vue
+++ b/packages/client/src/components/signup.vue
@@ -51,7 +51,7 @@
 				<span v-if="passwordRetypeState == 'not-match'" style="color: var(--error)"><i class="fas fa-exclamation-triangle fa-fw"></i> {{ $ts.passwordNotMatched }}</span>
 			</template>
 		</MkInput>
-		<MkSwitch v-if="meta.tosUrl" v-model="ToSAgreement" class="_formBlock tou">
+		<MkSwitch v-if="meta.tosUrl" v-model="ToSAgreement" class="_formBlock tou" style="border-color: rgb(215, 130, 126) !important;">
 			<I18n :src="$ts.agreeTo">
 				<template #0>
 					<a :href="meta.tosUrl" class="_link" target="_blank">{{ $ts.tos }}</a>
