diff --git a/packages/client/src/store.ts b/packages/client/src/store.ts
index 6ab14f43f..62e00b982 100644
--- a/packages/client/src/store.ts
+++ b/packages/client/src/store.ts
@@ -53,7 +53,7 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	reactions: {
 		where: 'account',
-		default: ['👍', '❤️', '😆', '🤔', '😮', '🎉', '💢', '😥', '😇', '🍮']
+		default: ['⭐', ':calc:', ':calc_wink:', ':calc_laugh:', ':calc_shocked:', ':calc_cold_sweat:', ':calc_swear:', ':calc_woozy:', ':calc_pleading:', ':calc_exhale:', ':calc_smile_hearts:', ':calc_kiss:', ':calc_headpats:', ':calc_innocent:', ':calcthonk:', ':calcflushed:', ':calchigh:', ':calcmentalbreakdowncrying:', ':calctrollage:', ':calcgunarm:', ':blobcat_attention:', ':amongass:', ':spamtroll:', ':flab:', ':sans_bonk:', ':ralseidead:']
 	},
 	mutedWords: {
 		where: 'account',
@@ -68,16 +68,20 @@ export const defaultStore = markRaw(new Storage('base', {
 		where: 'deviceAccount',
 		default: [
 			'notifications',
-			'favorites',
-			'drive',
+			'messaging',
 			'followRequests',
 			'-',
 			'featured',
 			'explore',
-			'announcements',
 			'search',
 			'-',
-			'ui',
+			'pages',
+			'gallery',
+			'groups',
+			'-',
+			'favorites',
+			'drive',
+			'clips'
 		]
 	},
 	visibility: {
@@ -151,11 +155,11 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	useBlurEffectForModal: {
 		where: 'device',
-		default: true
+		default: false
 	},
 	useBlurEffect: {
 		where: 'device',
-		default: true
+		default: false
 	},
 	showFixedPostForm: {
 		where: 'device',
@@ -183,15 +187,15 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	reactionPickerSize: {
 		where: 'device',
-		default: 1
+		default: 3
 	},
 	reactionPickerWidth: {
 		where: 'device',
-		default: 1
+		default: 3
 	},
 	reactionPickerHeight: {
 		where: 'device',
-		default: 2
+		default: 3
 	},
 	reactionPickerUseDrawerForMobile: {
 		where: 'device',
@@ -266,7 +270,7 @@ export class ColdDeviceStorage {
 		plugins: [] as Plugin[],
 		mediaVolume: 0.5,
 		sound_masterVolume: 0.3,
-		sound_note: { type: 'syuilo/down', volume: 1 },
+		sound_note: { type: 'none', volume: 0 },
 		sound_noteMy: { type: 'syuilo/up', volume: 1 },
 		sound_notification: { type: 'syuilo/pope2', volume: 1 },
 		sound_chat: { type: 'syuilo/pope1', volume: 1 },
