diff --git a/packages/client/src/store.ts b/packages/client/src/store.ts
index e6a2f4290..db455473a 100644
--- a/packages/client/src/store.ts
+++ b/packages/client/src/store.ts
@@ -53,7 +53,7 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	reactions: {
 		where: 'account',
-		default: ['👍', '❤️', '😆', '🤔', '😮', '🎉', '💢', '😥', '😇', '🍮']
+		default: ['👍', '❤️', '😆', '🤔', '😮', '😥', '😇', ':calchigh:', ':calcmentalbreakdowncrying:', ':amongass:']
 	},
 	mutedWords: {
 		where: 'account',
@@ -68,17 +68,19 @@ export const defaultStore = markRaw(new Storage('base', {
 		where: 'deviceAccount',
 		default: [
 			'notifications',
-			'messaging',
-			'drive',
 			'followRequests',
+			'messaging',
 			'-',
-			'gallery',
 			'featured',
 			'explore',
-			'announcements',
 			'search',
 			'-',
-			'ui',
+			'pages',
+			'gallery',
+			'-',
+			'favorites',
+			'drive',
+			'clips',
 		]
 	},
 	visibility: {
@@ -152,11 +153,11 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	useBlurEffectForModal: {
 		where: 'device',
-		default: true
+		default: false
 	},
 	useBlurEffect: {
 		where: 'device',
-		default: true
+		default: false
 	},
 	showFixedPostForm: {
 		where: 'device',
@@ -180,7 +181,7 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	instanceTicker: {
 		where: 'device',
-		default: 'remote' as 'none' | 'remote' | 'always'
+		default: 'always' as 'none' | 'remote' | 'always'
 	},
 	reactionPickerSize: {
 		where: 'device',
@@ -264,7 +265,7 @@ export class ColdDeviceStorage {
 		plugins: [] as Plugin[],
 		mediaVolume: 0.5,
 		sound_masterVolume: 0.3,
-		sound_note: { type: 'syuilo/down', volume: 1 },
+		sound_note: { type: 'syuilo/down', volume: 0 },
 		sound_noteMy: { type: 'syuilo/up', volume: 1 },
 		sound_notification: { type: 'syuilo/pope2', volume: 1 },
 		sound_chat: { type: 'syuilo/pope1', volume: 1 },
