diff --git a/packages/client/src/pages/welcome.entrance.a.vue b/welcome.entrance.a.vue
index b78a37eab..8b017bc4e 100644
--- a/packages/client/src/pages/welcome.entrance.a.vue
+++ b/welcome.entrance.a.vue
@@ -7,11 +7,20 @@
                <div class="shape2"></div>
                <img src="/client-assets/misskey.svg" class="misskey"/>
                <div class="emojis">
-                       <MkEmoji :normal="true" :no-style="true" emoji="👍"/>
-                       <MkEmoji :normal="true" :no-style="true" emoji="❤"/>
-                       <MkEmoji :normal="true" :no-style="true" emoji="😆"/>
-                       <MkEmoji :normal="true" :no-style="true" emoji="🎉"/>
-                       <MkEmoji :normal="true" :no-style="true" emoji="🍮"/>
+                       <MkEmoji :normal="true" emoji=":calc:"/>
+                       <MkEmoji :normal="true" emoji=":calc_wink:"/>
+                       <MkEmoji :normal="true" emoji=":calc_laugh:"/>
+                       <MkEmoji :normal="true" emoji=":calc_shocked:"/>
+                       <MkEmoji :normal="true" emoji=":calc_cold_sweat:"/>
+                       <MkEmoji :normal="true" emoji=":calc_swear:"/>
+                       <MkEmoji :normal="true" emoji=":calc_woozy:"/>
+                       <MkEmoji :normal="true" emoji=":calc_pleading:"/>
+                       <MkEmoji :normal="true" emoji=":calc_exhale:"/>
+                       <MkEmoji :normal="true" emoji=":calc_smile_hearts:"/>
+                       <MkEmoji :normal="true" emoji=":calc_kiss:"/>
+                       <MkEmoji :normal="true" emoji=":calc_headpats:"/>
+                       <MkEmoji :normal="true" emoji=":calc_innocent:"/>
+                       <MkEmoji :normal="true" emoji=":calc_flushed:"/>
                </div>
                <div class="main">
                        <img :src="$instance.iconUrl || $instance.faviconUrl || '/favicon.ico'" alt="" class="icon"/>
@@ -28,6 +37,7 @@
                                <div class="action">
                                        <MkButton inline rounded gradate data-cy-signup style="margin-right: 12px;" @click="signup()">{{ i18n.ts.signup }}</MkButton>
                                        <MkButton inline rounded data-cy-signin @click="signin()">{{ i18n.ts.login }}</MkButton>
+                                       <MkButton inline rounded onclick="window.location.href='https://stop.voring.me/featured'">Browse as guest</MkButton>
                                </div>
                        </div>
                </div>
